package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Base ESLint styles by Underdog.io",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint -c ./index.js ./**/*.js",
+    "lint": "npm run lint-es5 && npm run lint-es6",
+    "lint-es5": "eslint -c ./index.js ./index.js ./test/index.js",
+    "lint-es6": "eslint -c ./es6.js ./**/*.js",
```

Comment:
It would cause an issue if we wanted to add separate rules for things like block spacing or indenting